crates/compiler/types/src/subs.rs

```diff
@@ -50,10 +51,24 @@ impl fmt::Debug for Mark {
     }
 }
 
-#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
-pub enum ErrorTypeContext {
-    None,
-    ExpandRanges,
+bitflags! {
+    pub struct ErrorTypeContext : u8 {
+        const NONE = 1 << 0;
```

Comment:
Doesn't seem like we want to preserve `NONE` as its own bitflags, we get that for free when the other flags are zero.